todo_app/mongo_cards.py

```diff
@@ -0,0 +1,58 @@
+from typing import Collection
+import requests
+import os
+from dotenv import load_dotenv
+from requests.api import delete, request
+import pymongo
+import pprint
+from bson.objectid import ObjectId
+
+
+def get_items():
+    """ Simple attempt to get all cards from Trello. """
+    client  = pymongo.MongoClient(os.getenv('connection_string'))
+    database = client[os.getenv('database')]
+    collection = database.test_collection   
+    return collection.find()
+
+def get_done_items():
+    client  = pymongo.MongoClient(os.getenv('connection_string'))
+    database = client[os.getenv('database')]
+    collection = database.test_collection
+    return collection.find({"status": "done"})
+     
+def get_doing_items():
+    client  = pymongo.MongoClient(os.getenv('connection_string'))
+    database = client[os.getenv('database')]
+    collection = database.test_collection
+    return collection.find({"status": "doing"})
+     
+def get_todo_items():
+    client  = pymongo.MongoClient(os.getenv('connection_string'))
+    database = client[os.getenv('database')]
+    collection = database.test_collection
+    return collection.find({"status": "todo"})
+
+def add_new_item(name, desc):
+    client  = pymongo.MongoClient(os.getenv('connection_string'))
+    database = client[os.getenv('database')]
+    collection = database.test_collection
+    collection.insert_one({"name":name, "desc":desc, "status":"Todo"})
+    # collection.insert_one({"name":"Todo", "desc":"Module 5 Exercise", "status":"Todo"})
+    # collection.insert_one({"name":"Doing", "desc":"Module 4 Exercise", "status":"Doing"})
+    # collection.insert_one({"name":"Doing", "desc":"Module 3 Exercise", "status":"Doing"})
+    # collection.insert_one({"name":"Done", "desc":"Module 2 Exercise", "status":"Done"})
+    # collection.insert_one({"name":"Done", "desc":"Module 1 Exercise", "status":"Done"})
+
+
+def update_item(id):
+    client  = pymongo.MongoClient(os.getenv('connection_string'))
+    database = client[os.getenv('database')]
+    collection: Collection = database.test_collection
+    return collection.find_one_and_update({"_id": ObjectId(id)}, {"$set":{"status":"Done"}})
+
+def delete_item(id):
+    client  = pymongo.MongoClient(os.getenv('connection_string'))
+    database = client[os.getenv('database')]
+    collection = database.test_collection
+    return collection.delete({"_id": ObjectId(id)})
```

Comment:
This method errors, I think you want;
```suggestion
    return collection.delete_one({"_id": ObjectId(id)})
```